digdag-standards/src/main/java/io/digdag/standards/operator/jdbc/CsvWriter.java

```diff
@@ -2,15 +2,16 @@
 
 import java.io.IOException;
 import java.io.Writer;
+import java.io.Closeable;
 import java.util.List;
 import java.util.concurrent.Exchanger;
 
-public class CSVWriter
-    implements AutoCloseable
+public class CsvWriter
+    implements Closeable
```

Comment:
no. `close()` method can be declared to throw IOException even if the class implements AutoCloseable.
Maybe the reason is that Closeable.close is guaranteed be idempotent (can be called multiple times without exceptions) but AutoCloseable.close isn't required to be idempotent, according to [JavaDoc](https://docs.oracle.com/javase/8/docs/api/java/lang/AutoCloseable.html).